threedi_modelchecker/config.py

```diff
@@ -814,6 +814,15 @@ def is_none_or_empty(col):
             models.AggregationSettings.aggregation_method,
         ),
     ),
+    QueryCheck(
+        error_code=325,
+        level=CheckLevel.WARNING,
+        column=models.GlobalSetting.interception_global,
+        invalid=Query(models.GlobalSetting).filter(
+            ~is_none_or_empty(models.GlobalSetting.interception_file),
+        ),
+        message="v2_global_settings.interception_global should be defined also when using an interception_file.",
```

Comment:
`v2_global_settings.interception_global is recommended as fallback value when using an interception_file`